tests/core-addon/integration/core_addon.js

```diff
@@ -4,14 +4,15 @@
 
 const firefox = require("selenium-webdriver/firefox");
 const { Builder, By, until } = require("selenium-webdriver");
-const assert = require("assert");
+// const assert = require("assert");
 
 const firefoxOptions = new firefox.Options();
 firefoxOptions.setPreference("xpinstall.signatures.required", false);
+firefoxOptions.setPreference("extensions.experiments.enabled", true);
 // Unset this to run the UI (useful for local testing).
 firefoxOptions.headless();
 
-// This is the path to Firefox Nightly on Ubuntu with the Mozilla PPA
+// This is the path to Firefox Nightly on Ubuntu with the Mozilla PPA.
 if (process.platform === "linux") {
   firefoxOptions.setBinary("/usr/bin/firefox-trunk");
 }

```

Comment:
I can't comment on line 20, so I'm commenting here: we should change: `describe("Selenium", function () {` to something more appropriate, like `describe("Core-Addon Onboarding", function () {` since that describes the suite.